src/conq/navigation_lib/tool_detector_interface.py

```diff
@@ -0,0 +1,344 @@
+import argparse
```

Comment:
Reminder to add the triple quote docstring at the top of this that explains that this is quasi-demo-code quasi-prod-code

Reply:
Done in [94405e9](https://github.com/UM-ARM-Lab/conq_python/pull/52/commits/94405e94ecf3d69f882137a8e19eb9db1d792a00)